Source/OxyPlot.SkiaSharp/SkiaRenderContext.cs

```diff
@@ -0,0 +1,762 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="SkiaRenderContext.cs" company="OxyPlot">
+//   Copyright (c) 2020 OxyPlot contributors
+// </copyright>
+// --------------------------------------------------------------------------------------------------------------------
+
+namespace OxyPlot.SkiaSharp
+{
+    using global::SkiaSharp;
+    using System;
+    using System.Collections.Generic;
+    using System.Linq;
+
+    /// <summary>
+    /// Implements <see cref="IRenderContext" /> based on SkiaSharp.
+    /// </summary>
+    public class SkiaRenderContext : IRenderContext, IDisposable
+    {
+        private SKPaint paint = new SKPaint();
+        private SKPath path = new SKPath();
+        private float dpiScale = 1;
```

Comment:
Is there a reason this should be private and set through a method rather than just an auto property?

Reply:
Not really... You are right, probably an auto property would be neater.